apps/condo/domains/user/utils/testSchema/index.js

```diff
@@ -20,15 +21,16 @@ const { RESET_USER_MUTATION } = require('@condo/domains/user/gql')
 const User = generateGQLTestUtils(UserGQL)
 const UserAdmin = generateGQLTestUtils(UserAdminGQL)
 
+const usAreaCodes = countryPhoneData[0].mobile_begin_with.filter(x => x.length === 3)
 const createTestEmail = () => ('test.' + getRandomString() + '@example.com').toLowerCase()
-const createTestPhone = () => faker.phone.phoneNumber('+79#########')
+const createTestPhone = () => faker.random.arrayElement([faker.phone.phoneNumber('+79#########'), `+1${faker.random.arrayElement(usAreaCodes)}${faker.phone.phoneNumber('#######')}`])
```

Comment:
looks like generator for random US phone number needs to be moved to separate named function in order to increase code readability